test/test_paper-self-gravitating-gas-dynamics.jl

```diff
@@ -114,6 +114,13 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "
       linf = [0.26361780693997594, 1.3908873830688688, 1.3908873830688688, 4.066701303607613],
       tspan = (0.0, 0.005), initial_refinement_level=8, amr_callback=TrivialCallback())
   end
+
+  @testset "elixir_eulergravity_eoc_test_p4est.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_eulergravity_eoc_test_p4est.jl"),
+      l2   = [0.0002456507718735638, 0.0003374236900407321, 0.00033742369004074064, 0.0007198398465056179],
+      linf = [0.0013653498212873316, 0.0018996803803537077, 0.0018996803803517093, 0.004384818684607161],
+      tspan = (0.0, 0.1))
+  end
```

Comment:
Once the elixir is moved to a different location, please also add this to the corresponding test file